scraper.rb

```diff
@@ -34,7 +40,7 @@ def scrape_list(url)
     data = {
       name: name,
       gender: gender,
-      faction: tds[2].text,
+      faction: tds[2].children.map { |s| s.text + " " }.join.tidy,
```

Comment:
It's a little odd to add the space to each element in turn like this, and then join with nothing. Would probably be clearer to join with a space instead, e.g.

```ruby
tds[2].children.map(&:text).join(" ").tidy
```